### PR TITLE
cleaned up output

### PR DESCRIPTION
**Pull request by vin0110**

### CODE REVIEW

File: haascli/stack.py [hunk 1]
```diff
@@ -140,8 +141,14 @@ def create(ctx, stack_name, config_file, parameter, wait):
 
         if bad_response(response):
             ctx.abort()
-        stack_id = response['StackId'] if 'StackId' in response else None
-        print("StackId:", str(stack_id))
+        if 'StackId' in response:
+            stack_id = response['StackId']
+            print("StackId:", stack_id)
+            logger.info('created stack %s (%d)', stack_name, stack_id)
+        else:
```
Comment (oxhead) [inline haascli/stack.py:150]: If there is no StackId, does it mean stack creation failed?  In that case, do we need to call ctx.abort()?

**Comment by vin0110** [comment]
@oxhead please review.

**Comment by vin0110** [comment]
@oxhead please review

**Comment by oxhead** [comment]
I mentioned a corner case before.  When stacked id is not returned, the ctx.abort() should be called.  However, current implementation just print out a warning message.  This might cause an exception at

```
 waiter.wait(StackName=stack_name)
```

**Pull request merged**